batch-nft-reveal/app/pages/create.tsx

```diff
@@ -36,7 +36,11 @@ function CreatePage(): JSX.Element {
         )
         tx = await contract.deployTransaction.wait()
       } catch (ex) {
-        ex.message ? setError(ex.message) : setError('Unsuccessful deployment')
+        ex.reason
+          ? setError(ex.reason)
+          : ex.message
+          ? setError(ex.message)
+          : setError('Unsuccessful deployment')
```

Comment:
Wouldn't this look better with if statements? It's somewhat hard to read now. What do you think?